jjb/dynamic/scale-ci_install_baremetal.yml

```diff
@@ -0,0 +1,307 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        #!/bin/bash
+        set -o pipefail
+        set -eux
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2
+        chmod 600 ${PRIVATE_KEY}
+        git clone https://github.com/redhat-performance/JetSki.git
```

Comment:
we can delete this line in favor of line 301 where github plugin is used to clone the jetski repo in the workspace.

Reply:
done